formatting/discordFormat.ts

```diff
@@ -50,7 +54,7 @@ class DiscordFormat {
      * @param {Game} - The game to create an embed field for.
      * @return {Object} - The embed field to add for this game.
      */
-    makeMatchupField(game) {
+    makeMatchupField(game: Game): any {
```

Comment:
`string`?

Reply:
It's not though... I should define an interface but I got lazy